frontend/src/scenes/feature-flags/featureManagementLogic.ts

```diff
@@ -0,0 +1,101 @@
+import { actions, afterMount, connect, kea, listeners, path, props, reducers, selectors } from 'kea'
+import { loaders } from 'kea-loaders'
+import { actionToUrl, urlToAction } from 'kea-router'
+import api from 'lib/api'
+import { Scene } from 'scenes/sceneTypes'
+import { teamLogic } from 'scenes/teamLogic'
+import { urls } from 'scenes/urls'
+
+import { Breadcrumb, Feature } from '~/types'
+
+import type { featureManagementLogicType } from './featureManagementLogicType'
+
+export interface FeatureManagementLogicProps {
+    id?: Feature['id']
+}
+export interface FeaturesResult {
+    results: Feature[]
+    count: number
+    next?: string | null
+    previous?: string | null
+}
+
+export const featureManagementLogic = kea<featureManagementLogicType>([
+    props({} as FeatureManagementLogicProps),
+    path(['scenes', 'features', 'featureManagementLogic']),
+    connect({
+        values: [teamLogic, ['currentTeamId']],
+    }),
+    actions({
+        setActiveFeatureId: (activeFeatureId: Feature['id']) => ({ activeFeatureId }),
+    }),
+    reducers({
+        activeFeatureId: [
+            null as Feature['id'] | null,
+            {
+                setActiveFeatureId: (_, { activeFeatureId }) => activeFeatureId,
+            },
+        ],
+    }),
+    loaders(({ values }) => ({
+        features: [
+            { results: [], count: 0, offset: 0 } as FeaturesResult,
+            {
+                loadFeatures: async () => {
+                    const response = await api.get(`api/projects/${values.currentTeamId}/features`)
```

Comment:
I'll be swapping out this raw URL for a proper entry in `api.ts` in the next incoming PR